roles/kustomize_deploy/tasks/execute_step.yml

```diff
@@ -192,17 +192,19 @@
              path_join | realpath
           }}
       block:
+        # We have to use plain `kubectl kustomize`: lookup would
+        # be executed locally, on the ansible-controller. In CI,
+        # that would be the zuul-executor, and we don't manage them,
+        # leading to a risk to either NOT have kubectl, or an old,
+        # deprecated version.
         - name: "Build kustomized content for {{ stage.path }}"
-          ansible.builtin.copy:
-            dest: "{{ _output }}"
-            mode: "0644"
-            content: >-
-              {{
-                lookup(
-                  'kubernetes.core.kustomize',
-                  dir=_chdir
-                )
-              }}
+          environment:
+            PATH: "{{ cifmw_path | default(ansible_env.PATH) }}"
+          ansible.builtin.shell:
+            chdir: "{{ _chdir }}"
+            cmd: >-
+              set -o pipefail;
+              kubectl kustomize > {{ _output }}
```

Comment:
```suggestion
          ansible.builtin.command:
            chdir: "{{ _chdir }}"
            cmd: kubectl kustomize -o {{ _output }}
```

Reply:
i was thinking about adding `creates` but probably we want to enforce the file creation each time we run the role, right?

Reply:
checking in a few, I want a green run first, molecule seems to be broken on that role.